controllers/managedocs_controller.go

```diff
@@ -106,6 +106,12 @@ const (
 	convergedDeploymentType                = "converged"
 	consumerDeploymentType                 = "consumer"
 	providerDeploymentType                 = "provider"
+	prometheusRemoteWriteSecretName        = "managed-ocs-prom-remote-write-secret"
+	prometheusRemoteWriteSecretKey         = "prom-remote-write-config-secret"
+	prometheusRemoteWriteIDKey             = "prom-remote-write-config-id"
+	promRemoteWriteRelabelConfigMapName    = "prom-remote-write-relabel-config"
+	rhobsRelableConfigMapKey               = "rhobs_relabel_config.json"
```

Comment:
They will change in terms of environments staging/production,
Redacted the values since i don't feel I should so add them upstream but are completely ok to be downstream hence a config map would do. These values are pretty public downstream.